src/Dev.Development.Ver.src-1.0.0DdV/Convert/convert.ts

```diff
@@ -0,0 +1,68 @@
+import { basename } from "https://deno.land/std@0.196.0/path/win32.ts";

```

Comment:
Must not depend on [deno.land modules](https://deno.land/std@0.196.0/path/win32.ts).

Reply:
Change the dependency in the future